Documentation/ExtensionArchitecture/Tutorials/Tea/ExtensionConfiguration.rst

```diff
@@ -0,0 +1,45 @@
+..  include:: /Includes.rst.txt
+
+..  index::
+    Tutorial Tea; Extension configuration
+..  _extbase_tutorial_tea_extension_configuration:
+
+===================
+Create an extension
+===================
+
+For an extension to be installable in TYPO3 it needs a file called
+:file:`composer.json`. You can read more about this file here:
+:ref:`files-composer-json`.
+
+A minimal :file:`composer.json` to get the extension up and running
+could look like this:
+
+..  tabs::
+
+    ..  group-tab:: Composer
+
+        ..  include:: /CodeSnippets/Tutorials/Tea/ComposerJsonSimplified.rst.txt
+
+    ..  group-tab:: Legacy
+
+        ..  include:: /CodeSnippets/Tutorials/Tea/ExtEmconf.rst.txt
+
+
+..  hint::
+    If the extension should also be available for legacy installations it
+    also needs a file called :file:`EXT:tea/ext_emconf.php`. This file
```

Comment:
```suggestion
    also needs a file called :file:`ext_emconf.php`. This file
```
Would avoid EXT:tea as it is also not mentioned above when mentioning composer.json.